Entities/Ingame/ClubFinances.cs

```diff
@@ -74,9 +74,17 @@ public ClubFinances (int memoryAddress, ArraySegment<byte> originalBytes, IVersi
 		}
 
 		public int WeeklyWageBudget {
-			get {
-				return PropertyInvoker.Get<Int32> (ClubFinancesOffsets.WeeklyWageBudget, OriginalBytes, MemoryAddress, DatabaseMode);
-			}
+			get {
+                int value = PropertyInvoker.Get<Int32>(ClubFinancesOffsets.WeeklyWageBudget, OriginalBytes, MemoryAddress, DatabaseMode);
+                int subValue = PropertyInvoker.Get<Int32>(0x4C, OriginalBytes, MemoryAddress, DatabaseMode);
+
+                if (value == 0 || subValue == 0 || subValue < 0 || subValue > 10000)
+                {
+                    return 0;
+                }
+                value = value + subValue;
+                return value;
+            }
 		}
```

Comment:
Ah, I see why you are confused.
In the game, the figure you see is called ```WeeklyWageBudget``` but in reality this is the sum of two different values.
The value at offset 0x4C is ```YouthGrantIncome```, it is not an encryption algorithm.
You can add this value and add a computed property if you want to match the game's output.